base/bitarray.jl

```diff
@@ -235,8 +235,11 @@ function fill!(B::BitArray, x)
     return B
 end
 
-falses(args...) = fill!(BitArray(args...), false)
-trues(args...) = fill!(BitArray(args...), true)
+for (f, v) in ((:falses, false), (:trues, true))
```

Comment:
Honestly, I don't think the `@eval` machinery is worth it here. It makes the code much harder to read.

Also, do you know if both variants are currently tested?